Assignment4/NumberSet/NumSet.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NumberSet
+{
+    public class NumSet
+    {
+        private HashSet<int>? numberSet;
+
+
+        public NumSet(params int[] ? numSet)

```

Comment:
```suggestion
        public NumSet(params int[]? numSet)
```
A quick keyboard command to format your code in Visual Studio is `ctrl+k ctrl+d`. You can also right-click on your solution file and select `Analyze and code cleanup` to do a solution-wide cleanup (with customized clean up options).